go.mod

```diff
@@ -116,5 +122,8 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.23.0
 	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.23.0
 	k8s.io/metrics => k8s.io/metrics v0.23.0
+	k8s.io/mount-utils => k8s.io/mount-utils v0.23.0
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.23.0
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.11.0
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.7.0
```

Comment:
why do you need these replaces? 
Why do you import sigs.k8s.io/controller-tools?